app/bundles/CoreBundle/Assets/css/app/less/components/alerts.less

```diff
@@ -14,6 +14,68 @@
   }
 }
 
+.alert {
+  border-width: 0 0 0 3px;
+  color: var(--text-primary);
+  display: flex;
+  flex-wrap: nowrap;
+  gap: 3px;
+  background-color: var(--layer);
+  border-radius: var(--border-radius-sm);
+
+  &.alert-success,
```

Comment:
You can also move this block of 4 alert classes outside, so it is not nested.

Reply:
I’ll proceed with these changes, but they can be problematic if we consider that they’re planned to avoid overwriting…

Some components should follow a standard that we don’t want to allow devs to replace and create something out of the initial consistency 

When thinking about Bootstrap as a framework, makes sense to avoid nesting things so they keep customizable. It’s not the same case when we have a specific set of design guidelines (or a design system): components should look the same everywhere and be limited to our already built options.

If extending is really necessary, then we should extend the component (alerts.less) to create something new instead of making this at the Twig/HTML level

Reply:
While I agree that we want to have a design system with guidelines, it's not the job of CSS to try and limit or block avoid customization. That's not what it's for. Nesting things is fine if it is functional but not to make it more difficult for someone to override styling. You don't just add complexity for complexity's sake or it will bite your or other developers later down the road. This is a really bad practice and true for all programming.

Reply:
Got it!